packages/sitecore-jss-nextjs/src/index.ts

```diff
@@ -1,13 +1,9 @@
 export {
   constants,
   // generic data access
-  HttpDataFetcher,
```

Comment:
This export need to remain - we still want to allow custom fetchers for REST services